backend/typescript/models/index.ts

```diff
@@ -1,13 +1,29 @@
 import * as path from "path";
 import { Sequelize } from "sequelize-typescript";
+import User from "./user.model";
+import AnimalType from "./animalType.model";
+import UserAnimalTypes from "./userAnimalTypes.model";
 
 const DATABASE_URL =
   process.env.NODE_ENV === "production"
     ? /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
       process.env.DATABASE_URL!
     : `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.DB_HOST}:5432/${process.env.POSTGRES_DB_DEV}`;
 
-/* eslint-disable-next-line import/prefer-default-export */
-export const sequelize = new Sequelize(DATABASE_URL, {
+const sequelize = new Sequelize(DATABASE_URL, {
   models: [path.join(__dirname, "/*.model.ts")],
 });
+
+/* eslint-disable-next-line import/prefer-default-export */
+sequelize.addModels([User, AnimalType, UserAnimalTypes]);
+
+User.belongsToMany(AnimalType, {
```

Comment:
is there a reason why we need to change this file?

Reply:
@aanxniee Yes unfortunately, I was receiving a circular dependency error. I spoke with Sayi and he told me that while the other people on the team had one, they weren't modelling a many to many relationship like in this PR. I tried to resolve them in their specific tables but it didn't work. The information I found online suggested having some kind of outside file to resolve all the relationships like an index file. I can try and find other solutions, I didn't want to edit the index file either

Reply:
hmm i see @laks0407 @jerry-cheng5